daemon/import.go

```diff
@@ -35,7 +35,7 @@ func (daemon *Daemon) ImportImage(src string, repository, tag string, msg string
 
 	if repository != "" {
 		var err error
-		newRef, err = reference.ParseNamed(repository)
+		newRef, err = reference.ParseNormalizedNamed(repository)
```

Comment:
Will also add an error here like the previous case, it should change to take in `reference.Named`